code/game/jobs/job/job.dm

```diff
@@ -163,10 +164,14 @@
 	. = outfit.equip(H, title, alt_title, OUTFIT_ADJUSTMENT_SKIP_POST_EQUIP|OUTFIT_ADJUSTMENT_SKIP_ID_PDA|additional_skips)
 
 /datum/job/proc/get_access()
+	var/list/returned_access = list()
 	if(minimal_access.len && (!config || config.jobs_have_minimal_access))
-		return src.minimal_access.Copy()
+		returned_access = src.minimal_access.Copy()
 	else
-		return src.access.Copy()
+		returned_access = src.access.Copy()
+	if(lowpop_access.len && (GLOB.clients.len < config.lowpop_access_amount) && (!config || config.lowpop_access))
```

Comment:
`&& (!config || config.lowpop_access)`

"...and there is either no config or the config has lowpop_access enabled" -- can you elaborate on what this part of the if statement means?

Reply:
Probably copy-paste from config code, they always use this.
tl;dr, if config somehow doesn't exist - it will assume lowpop_access is enabled.

Reply:
If that is the intention, then this would throw an error before that anyway, as trying to access `config.lowpop_access_amount` would result in an NRE.

If the intention is to have it default to on if there is no config, you'll need to rewrite it to short circuit it.